sites/partners/src/listings/PaperListingForm/UnitForm.tsx

```diff
@@ -173,6 +272,25 @@ const UnitForm = ({ onSubmit, onClose, units, currentTempId }: UnitFormProps) =>
     })
   }, [amiCharts, unitPriorities, unitTypes])
 
+  useEffect(() => {
+    if (defaultUnit) {
+      setValue("amiChart.id", defaultUnit.amiChart?.id)
+      setValue("priorityType.id", defaultUnit.priorityType?.id)
+      setValue("unitType.id", defaultUnit.unitType?.id)
+    }
+  }, [options])
```

Comment:
The linter is complaining about the dependency arrays for this and two others. Does updating them cause issues?